docsite/rst/intro_inventory.rst

```diff
@@ -231,8 +231,7 @@ SSH connection::
     ansible_ssh_extra_args
       This setting is always appended to the default ssh command line.
     ansible_ssh_pipelining
-      Determines whether or not to use SSH pipelining. This can override the
-      ``pipelining`` setting in ``ansible.cfg``.
+      Determines whether or not to use SSH pipelining. This can override the ``pipelining`` setting in ``ansible.cfg``.
```

Comment:
Unfortunately, you need to word wrap this and your other entries here as well.  The reason is that we'the wrong markup for these areas.  We're using ```::``` which means a code block or preformatted text.  So long lines are not going to wrap in the html display either.

We shold probably be using definition lists for all of these instead of code blocks.  As a short example that would look like:
```
:ansible_ssh_extra_args:
       This setting is always appended to the default ssh command line.
:ansible_ssh_pipelining: Determines whether or not to use SSH pipelining. This can override the ``pipelining`` setting in ``ansible.cfg``.
```

( The definition can start on the same line as the definition term or the next line.  If you have line breaks (I set mine at 160 chars in a compromise between you and sanity ;-) subsequent lines just need to be indented.

I'll leave it up to you whether you want to add line breaks or reformat the lists to be definition lists instead.